### DISCUSSION TITLE
URL scheme to start a VM in disposable mode

### DISCUSSION THREAD

**Discussion by orazioedoardo** [utmapp/UTM#4236] [Ideas]
[Looks like](https://github.com/utmapp/UTM/wiki/URL-Scheme-%E2%80%93-Automation-for-UTM-app-&-VMs) there isn't one.

**Comment by macttys** [comment]
This would be useful.  Possible syntax:

``utm://start_disposable?name=Ubuntu%2020.04``  or 
``utm://start?name=Ubuntu%2020.04&type=disposable``
